sbt-conductr-tester/lagom-scala-bundle/build.sbt

```diff
@@ -15,9 +15,6 @@ lazy val `play-service` = (project in file("play-service"))
   .enablePlugins(PlayScala, LagomPlay)
   .settings(
     routesGenerator := InjectedRoutesGenerator,
-    // Need to explicitly add the dependency to lagom-client 1.3.1 because 1.3.1-RC1 is being pulled in
```

Comment:
This seems not to be required any longer. 1.3.5 is correctly evicting the RC.

In fact, I think this was true as of 1.3.1 (there were no RCs for 1.3.1, only 1.3.0). I'm guessing the comment was updated via an automatic search and replace.